aws/resource_aws_lightsail_instance.go

```diff
@@ -233,6 +245,19 @@ func resourceAwsLightsailInstanceDelete(d *schema.ResourceData, meta interface{}
 	return nil
 }
 
+func resourceAwsLightsailInstanceUpdate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).lightsailconn
+
+	if d.HasChange("tags") && !d.IsNewResource() {
```

Comment:
The resource `Create` function does not call `Update`, so the `d.IsNewResource()` check is extraneous. 👍 

```suggestion
	if d.HasChange("tags") {
```

Reply:
Yeah, copy/paste error, sorry.